app/controllers/application_controller.rb

```diff
@@ -5,12 +5,15 @@ class ApplicationController < ActionController::Base
 
   def set_locale
     locale = params[:locale].to_s.strip.to_sym || I18n.default.locale
-    I18n.locale = I18n.available_locales.include?(locale) ?
-      locale : I18n.default_locale
+    lang = if I18n.available_locales.include?(locale)
+             locale
+           else
+             I18n.default_locale
+           end
+    I18n.locale = lang
```

Comment:
Thế sao không fix như này
```suggestion
   condition = I18n.available_locales.include?(locale)
   I18n.locale = condition ? locale : I18n.default_locale
```

Reply:
Fix như này nhìn gọn hơn ạ. Em update lại ạ